wfe2/stats.go

```diff
@@ -15,6 +15,9 @@ type wfe2Stats struct {
 	// csrSignatureAlgs counts the signature algorithms in use for order
 	// finalization CSRs
 	csrSignatureAlgs *prometheus.CounterVec
+	// improperECFieldLengths counts the number of EC keys we see with
```

Comment:
same nit here RE: specifying this is specific to account keys not anything in CSRs, etc.

Reply:
Resolved in 8a718ef